src/main/java/no/digipost/api/client/internal/ApiServiceImpl.java

```diff
@@ -601,7 +603,23 @@ private CloseableHttpResponse sendDigipostMedia(Object data, String uri) {
         httpPost.setHeader(Content_Type_DIGIPOST_MEDIA_TYPE_V8);
         ByteArrayOutputStream bao = new ByteArrayOutputStream();
         marshal(jaxbContext, data, bao);
-        httpPost.setEntity(new ByteArrayEntity(bao.toByteArray()));
+        httpPost.setEntity(new ByteArrayEntity(bao.toByteArray(), ContentType.create(DIGIPOST_MEDIA_TYPE_V8)));
         return send(httpPost);
     }
+
+    private HttpClientResponseHandler<CloseableHttpResponse> responseHandler() {
```

Comment:
Hva er dette for noe? Fins det ikke noen defaults i http-client som gjør denne jobben?